### DISCUSSION TITLE
Decrease speed when reach 50%

### DISCUSSION THREAD

**Discussion by passkordum** [TheLastBen/fast-stable-diffusion#2170] [Q&A]
I have an issue, when my generation process hit 50% it slow down, however, before 50% it generate very fast, what it can be?

**Comment by TheLastBen** [comment]
probably you're using highres.fix, from 50% it starts generating at a higher resolution

**Reply by passkordum** [reply]
it starts to slow down the process, when colab changes types of available GPUs, before it, it was fast enough, is it possible to make it faster after this colab update?
